lib/cfer/core/stack.rb

```diff
@@ -159,6 +159,17 @@ def to_cfn
       to_h.to_json
     end
 
+    # Includes template code from one or more files, and evals it in the context of this stack.
+    # Filenames are relative to the file containing the invocation of this method.
+    def include_template(*files)
+      calling_file = caller.first.split(/:\d/,2).first
```

Comment:
@seanedwards Sooooo, this is a most heinous hack, but I can't think of a better way to get the filename of the base template file. I copied it from a much better programmer than I (https://github.com/steveklabnik/require_relative/blob/master/lib/require_relative.rb#L24), so perhaps that gives it a little more respectability ...